client-app/src/desktop/tabs/containers/TabPanelContainerPanel.js

```diff
@@ -42,6 +42,7 @@ export const tabPanelContainerPanel = hoistCmp.factory({
                 height: 400,
                 item: tabContainer({
                     model: {
+                        persistWith: {localStorageKey: 'toolboxContainerTabs'},
```

Comment:
maybe `tabExampleState`? -- toolbox is going to be implied.  Also have been using the 'state' suffix to denote persistWith prefs and localStorage

Reply:
Suggestion applied.